src/optimizer/query_planner.c

```diff
@@ -4217,18 +4217,17 @@ qo_plan_cmp (QO_PLAN * a, QO_PLAN * b)
     {
       return PLAN_COMP_EQ;
     }
-  if (af <= bf && aa <= ba)
+  if (af + aa <= bf + ba)
```

Comment:
Is it correct that the case of returning the PLAN_COMP_UNK disappears? It is a structure that is determined unconditionally by either side.

In the current comparison method, `af (fixed cost)` is treated the same as `aa (variable cost)`. Don't you need to weigh it?

Reply:
Did you read my comments on Teams? Please ask specific questions about what you do not understand or have questions about.